integration_test/EchoNotificationTest.re

```diff
@@ -8,7 +8,7 @@ runTest(~name="InputIgnore test", (_dispatch, wait, runEffects) => {
   );
 
   // Create notification - an echo should trigger onj
-  Vim.command("echo 'hi from test'");
+  Vim.command("echo 'hi from test'") |> ignore;
```

Comment:
These are all accidental partial application bugs waiting to happen.

```suggestion
  ignore(Vim.command("echo 'hi from test'"): Vim.Context.t);
```

Reply:
Good point!

I started factoring these to be an action:

```
dispatch(Actions.VimExecuteCommand("echo 'hi from test'"))
```

It's not an ideal action, because it's very function-call-ish. But I realized we would need an effect handler to handle this, because things like `Vim.command("echo 'hi from test'")` will have effects that we'll need to map and react to - so it made sense to centralize that logic.